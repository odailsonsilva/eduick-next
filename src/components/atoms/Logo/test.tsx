import { screen } from '@testing-library/react'
import { renderWithTheme } from '../../../utils/tests/helpers'

import { Logo } from '.'

describe('<Logo />', () => {
  it('shold render defaul logo', () => {
    renderWithTheme(<Logo />)
    expect(screen.getByLabelText('logo')).toBeInTheDocument()
    // expect(container.firstChild).toMatchSnapshot()
  }),
    it('shold render defaul logo with detail', () => {
      const { container } = renderWithTheme(<Logo activedDetails />)
      expect(screen.getByLabelText('logo')).toBeInTheDocument()
      expect(screen.getByLabelText('detailLogo')).toBeInTheDocument()
      expect(container.firstChild).toMatchSnapshot()
    })
})
