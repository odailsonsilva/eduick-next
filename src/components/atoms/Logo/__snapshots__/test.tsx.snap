// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> shold render defaul logo with detail 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 > div + div {
  margin-left: 2.2rem;
}

<div
  class="c0"
>
  <div
    aria-label="detailLogo"
  >
    <div>
      <div
        data-src="/logos/details-logo.svg"
        src=""
      />
    </div>
  </div>
  <div
    aria-label="logo"
  >
    <div>
      <div
        data-src="/logos/logo.svg"
        src=""
      />
    </div>
  </div>
</div>
`;
